index.html

```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
+         </p>
+         <ul class="tools">
+          <li class="tools-items">HTML</li>
+          <li class="tools-items">CSS</li>
+          <li class="tools-items">JAVASCRIPT</li>
+         </ul>
+         <button class="user-buttons">
+           See Project
+         </button>
+        </footer>
+      </section>
+    </section>
+   <!-- Work section starts -->
+
+   <!-- About Myself starts -->
+
+  <section class="about-headline">
+    <section class="about-headline-heading">
+        <h1>About Myself</h1>
+    </section>
+    <section class="about-headline-description">
+        <p>
+                I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don't hesitate to contact me.
+        </p>
+    </section>
+    <section class="about-social-media">
+        <h2>LET'S CONNECT</h2>
+    </section>
+      <ul class="about-social-media-icons">
+        <li>
+          <a href="#">
+            <img src="./Properties/Twitter.png" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Linkedin icon.png" alt="linkedin">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Mail.png" alt="mail">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Github.png" alt="github">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Vector.png" alt="vector">
+          </a>
+        </li>
+      </ul>
+      <button class="user-buttons">
+        Get my resume
+      </button>
+
+      <section class="languages-frameworks">
+        <ul class="languages-frameworks-list">
+          <li>
+            <article class="language-title">
+              <h2>Languages</h2>
+              <img src="./Properties/down-arrow.svg" alt="">
+            </article>
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/js.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
+              </section>
+            </article>
+            
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/html.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
+              </section>
+            </article>
+
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/css.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
```

Comment:
- [ ] I would suggest you please list each category(e.g., "JavasScript", "HTML" and "CSS") in ul and each of them a li for better semantic HTML.